data-prepper-pipeline-parser/build.gradle

```diff
@@ -12,7 +12,23 @@ group = 'org.opensearch.dataprepper.core'
 dependencies {
     implementation project(':data-prepper-api')
     implementation project(':data-prepper-plugins:blocking-buffer')
-    implementation 'com.fasterxml.jackson.core:jackson-databind'
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.12.3'
     implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'
     implementation 'org.apache.commons:commons-collections4:4.4'
+    implementation 'org.apache.commons:commons-jexl3:3.2.1'
+    implementation 'org.projectlombok:lombok:1.18.22'
+    implementation 'com.jayway.jsonpath:json-path:2.6.0'
+    implementation 'javax.inject:javax.inject:1'
+    implementation(libs.spring.core) {
+        exclude group: 'commons-logging', module: 'commons-logging'
+    }
+    implementation(libs.spring.context) {
+        exclude group: 'commons-logging', module: 'commons-logging'
+    }
+    implementation 'software.amazon.cloudwatchlogs:aws-embedded-metrics:2.0.0-beta-1'
```

Comment:
Do you need this here?

Reply:
removed ones i am not using. addressed in the latest pr